fpga/pulpissimo-zedboard/Makefile

```diff
@@ -1,9 +1,8 @@
-PROJECT:=pulpissimo-zedboard
-VIVADO ?= vivado
-VIVADOFLAGS ?= -nojournal -mode batch -source scripts/prologue.tcl
-
 include fpga-settings.mk
```

Comment:
Is there a reason why you remove the default `VIVADOFLAGS`?

Reply:
I am not even convinced that `vivado` picks that variable up. I could not find any documentation on that neither any mention of it in the Xilinx forums, and the following does not seem to indicate it obeys anything specified in it:

    $ export VIVADOFLAGS="-nojournal -mode batch"
    $ vivado
    ****** Vivado v2018.3 (64-bit)
      **** SW Build 2405991 on Thu Dec  6 23:36:41 MST 2018
      **** IP Build 2404404 on Fri Dec  7 01:43:56 MST 2018
        ** Copyright 1986-2018 Xilinx, Inc. All Rights Reserved.

    start_gui
    […]
    $ la
    total 8
    -rw-r--r-- 1 stefanct stefanct 448 Jan 22 13:47 vivado.jou
    -rw-r--r-- 1 stefanct stefanct 448 Jan 22 13:47 vivado.log

Additionally, there is no file named `prologue.tcl` (and never was) in any of the related repositories AFAICT. So I am inclined to reply with a counter question: why was it added everywhere? ;)
In any case I could not see any differences after removing it.